### PR TITLE
modify file loss-function.ipynb

### PR DESCRIPTION
**Pull request by WHQWHQWHQ**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How has this been tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] [local build the book](https://github.com/ocademy-ai/machine-learning/blob/main/CONTRIBUTING.md#build-the-book)
- [ ] review the artifacts from local build
- [ ] [local build the slides](https://github.com/ocademy-ai/machine-learning/blob/main/CONTRIBUTING.md#build-the-slides-optional) if appliable
- [ ] Unit test if appliable

# Checklist:

- [ ] My code follows the [style guidelines](https://github.com/ocademy-ai/machine-learning/blob/main/STYLE_GUIDE.md) of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have applied tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: open-machine-learning-jupyter-book/ml-fundamentals/regression/loss-function.ipynb [hunk 1]
```diff
@@ -133,14 +136,15 @@
    "source": [
     "from sklearn.preprocessing import StandardScaler\n",
     "\n",
-    "X = df_stock.drop(columns=['Close_forcast'], axis=1)\n",
-    "y = df_stock['Close_forcast']\n",
+    "X = df_stock.drop(columns=[\"Close_forcast\"], axis=1)\n",
+    "y = df_stock[\"Close_forcast\"]\n",
     "\n",
     "scaler_X = StandardScaler()\n",
     "X = scaler_X.fit_transform(X)\n",
```
Comment (Lola-jo) [inline open-machine-learning-jupyter-book/ml-fundamentals/regression/loss-function.ipynb:143]: This line of code doesn't run, does it?

Comment (WHQWHQWHQ) [reply open-machine-learning-jupyter-book/ml-fundamentals/regression/loss-function.ipynb:147]: Done

File: open-machine-learning-jupyter-book/ml-fundamentals/regression/loss-function.ipynb [hunk 2]
```diff
@@ -57,7 +57,7 @@
    "id": "f5ee754a-e5d9-4532-86d5-1433722be122",
```
Comment (Lola-jo) [inline open-machine-learning-jupyter-book/ml-fundamentals/regression/loss-function.ipynb:57]: <p>I think a main heading should be added, and then this heading should be changed to a secondary heading. </p><p>And the same operation should be done for all the following primary headings.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/590/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (Lola-jo) [inline open-machine-learning-jupyter-book/ml-fundamentals/regression/loss-function.ipynb:57]: <p>Replace "you" with "we".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/590/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='72'/>

Comment (Lola-jo) [inline open-machine-learning-jupyter-book/ml-fundamentals/regression/loss-function.ipynb:57]: <p>There should be a 'your turn' and 'acknowledge' section after that.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/590/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='74'/>
